processing/src/main/java/org/apache/druid/query/aggregation/AggregatorUtil.java

```diff
@@ -100,7 +100,9 @@
   public static final byte ARRAY_OF_DOUBLES_SKETCH_TO_QUANTILES_SKETCH_CACHE_TYPE_ID = 0x28;
   public static final byte ARRAY_OF_DOUBLES_SKETCH_T_TEST_CACHE_TYPE_ID = 0x29;
   public static final byte ARRAY_OF_DOUBLES_SKETCH_TO_STRING_CACHE_TYPE_ID = 0x2A;
-
+  public static final byte ARRAY_OF_DOUBLES_SKETCH_CONSTANT_SKETCH_CACHE_TYPE_ID = 0x4D;
```

Comment:
I think it would be better to append these at the last in a separate section in increasing hex value order. This makes it hard to know the next available id and can very possibly lead to collisions.

Reply:
done